### ISSUE TITLE
E684: List index out of range: 0

### ISSUE THREAD

**Issue by VimWei** [lervag/wiki.vim#380]
# Description

After upgrading wiki.vim and restarting gvim, gvim displays the following error:

```
Messages maintainer: The Vim Project
Error detected while processing C:/Users/chenw/vimfiles/plugged/wiki.vim/plugin/wiki.vim:
line  138:
E684: List index out of range: 0
line  142:
E121: Undefined variable: s:default_viewer
E116: Invalid arguments for function wiki#init#option
```

![image](https://github.com/user-attachments/assets/b16b0190-d3d2-413b-bdbb-fc71bb605ba6)

- OS:  Win 10 pro
- gvim: 9.1.697

**Comment by lervag** [comment]
Oh, sorry, that's my mistake! I'll fix it asap!

**Comment by lervag** [comment]
I think it's fixed now with commit ba630e2e.

**Comment by VimWei** [comment]
Thanks! It's fixed.
